types/lodash/index.d.ts

```diff
@@ -10158,7 +10158,7 @@ declare namespace _ {
          * @param value The value to check.
          * @returns Returns true if value is correctly classified, else false.
          */
-        isWeakMap(value?: any): value is WeakMap<any, any>;
+        isWeakMap<K extends object, V>(value?: any): value is WeakMap<K, V>;
```

Comment:
Would it work to just make this something like `isWeakMap(value?: any): value is WeakMap<object, any>`?

Reply:
Done!